src/main/java/ca/corefacility/bioinformatics/irida/ria/web/components/datatables/DataTablesExportToFile.java

```diff
@@ -71,26 +86,32 @@ private static void writeToExcel(HttpServletResponse response, String filename,
 		}
 
 		response.setContentType("application/vnd.ms-excel");
-		response.setHeader("Content-disposition",
-				"attachment; filename=" + filename + ".xlsx");
+		response.setHeader("Content-disposition", "attachment; filename=" + filename + ".xlsx");
 		workbook.write(response.getOutputStream());
+
+		workbook.close();
```

Comment:
Added in close of workbook, which was reported as not being closed in compiler warning.